include/rocksdb/env.h

```diff
@@ -740,6 +740,14 @@ class RandomAccessFile {
   // RandomAccessFileWrapper too.
 };
 
+// A data structure brings the data verification information, which is
+// used togther with data being written to a file.
+struct DataVerificationInfo {
+  DataVerificationInfo(const std::string& checksum_) : checksum(checksum_) {}
```

Comment:
Can you explaining the reason of constructing checksum using a string? Should constructing Slice using a Slice better?